starter/source/output/th/hm_read_prethgrou.F

```diff
@@ -140,55 +145,43 @@ SUBROUTINE HM_READ_PRETHGROU(IFI,NVARTOT,LSUBMODEL,IFLAG)
         THFILE = '/ITH'
       ENDIF
 C
-C-----------------------------------------------      
-!     **************************    !
-!       /TH read with hm reader     !
-!     **************************    !
-C-----------------------------------------------
       !-------------------------------------------
       ! /TH/ACCEL
       !-------------------------------------------
       CALL HM_OPTION_COUNT(TRIM(THFILE)//'/ACCEL',NTHACCEL)
-      IF (NTHACCEL > 0) THEN 
-        CALL HM_OPTION_START(TRIM(THFILE)//'/ACCEL')
```

Comment:
Removing this , will not initiate the pointer to the option , before reading with HM_OPTION_READ_KEY